src/ruby/ext/grpc/rb_grpc.c

```diff
@@ -192,6 +192,10 @@ static gpr_timespec zero_realtime;
 static gpr_timespec inf_future_realtime;
 static gpr_timespec inf_past_realtime;
 
+static void Init_grpc_logging() {
+  gpr_log_init();
```

Comment:
I added this because it makes it so that `require 'grpc'` in Ruby will properly set the macOS time zone automatically, but this might be considered a leaky abstraction since `grpc_init()` already executes `gpr_log_init()`.